packages/frontend/src/features/core/utils/genReportIdentityProof.ts

```diff
@@ -0,0 +1,62 @@
+import { ReportIdentityProof } from '@unirep-app/circuits'
+import { UserState } from '@unirep/core'
+import { toDecString } from '@unirep/core/src/Synchronizer'
+import { stringifyBigInts } from '@unirep/utils'
+import { poseidon2 } from 'poseidon-lite'
+
+export function genReportNullifier(id: bigint, reportId: string) {
```

Comment:
the parameter `id` can be changed to `id_secret` (just want to clarify what the parameter really means)

Reply:
fixed